src/task/CamelKTaskCompletionItemProvider.ts

```diff
@@ -22,65 +22,66 @@ import { TraitManager } from './TraitManager';
 
 export class CamelKTaskCompletionItemProvider implements vscode.CompletionItemProvider {
 
-    provideCompletionItems(document: vscode.TextDocument, position: vscode.Position, token: vscode.CancellationToken, context: vscode.CompletionContext): vscode.ProviderResult<vscode.CompletionItem[] | vscode.CompletionList> {
-        return this.provideCompletionItemsForText(document.getText(), document.offsetAt(position));
-    }
+	provideCompletionItems(document: vscode.TextDocument, position: vscode.Position, token: vscode.CancellationToken, context: vscode.CompletionContext): vscode.ProviderResult<vscode.CompletionItem[] | vscode.CompletionList> {
```

Comment:
Parameters  token: vscode.CancellationToken  and context: vscode.CompletionContext are not used in the function and could be removed to clean up the code.

Reply:
cannot be removed as it is implementing an API

Reply:
ah ok, didn't check that.